mapping/sustainability.yaml

```
@@ -1714,16 +1714,17 @@
 
         For each meeting:
 
-        1. Publish the meeting agenda. Add a document, set its `.documentType` to 'lobbyingMeetingAgenda' and its `.url` to the URL at which the agenda is available.
-
-        2. Publish the meeting details. Add a `Meeting` object to the `.lobbyingMeetings` array and set:
+        1. Publish the meeting details. Add a `Meeting` object to the `.lobbyingMeetings` array and set:
```

Explain what to do if the public official's organization doesn't exist yet. Copy-edit to avoid referencing the public official's organization before it is introduced (GitHub won't let me add a multi-line comment today so you'll also need to delete the subsequent lines).

```suggestion
        1. Publish the meeting details:
          1. Add a `Meeting` object to the `.lobbyingMeetings` array and set:
            - `.id` incrementally
            - `.date` to the date of the meeting
            - `.address` to the address of the meeting
            - `.numberOfParticipants` to the number of people present at the meeting
            - `.publicOfficial.name` to the name of the public official present at the meeting
            - `.publicOfficial.jobTitle` to the job title of the public official present at the meeting
          1. Get the `Organization` in `.parties` that represents the public official's organization. If none exists yet, [add an organization](../common.md#add-an-organization).
          1. Set the meeting's `.publicOfficial.organization` to the `.id` and `.name` of the organization.  
```